client/components/nux-notice/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { translate as __ } from 'i18n-calypso';
+
+const NuxNotice = ( { children, onDismiss } ) => {
```

Comment:
Oh, now I see the dismissal function comes from the parent component anyway. Yeh, I definitely think we should move this pack to the parent component.